source/BoardingPanel.cpp

```diff
@@ -310,6 +323,7 @@ bool BoardingPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 	{
 		int yourStartCrew = you->Crew();
 		int enemyStartCrew = victim->Crew();
+		int startShares = Crew::SharesForShip(player.FlagshipPtr(), true, true);
```

Comment:
```suggestion
```
With the above suggestion, this line of code becomes unnecessary because we won't be calculating profit shares in this file.